src/Idris/IdeMode.hs

```diff
@@ -205,28 +206,31 @@ escape = concatMap escapeChar
     escapeChar '"'  = "\\\""
     escapeChar c    = [c]
 
+type Parser a = P.Parsec () String a
+
+pSExp :: Parser SExp
```

Comment:
The convention has usually been just using the name without prefixing with `p`.
I know you did not add the change, but I just discovered it now.

I would prefer if this was called `sexp` or similar.

Reply:
OK, will do.